src/sentry/static/sentry/app/views/dashboardsV2/widget/metricWidget/queries.tsx

```diff
@@ -1,65 +1,104 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
+// import capitalize from 'lodash/capitalize';
+// import memoize from 'lodash/memoize';
 import Button from 'app/components/button';
-import SmartSearchBar from 'app/components/smartSearchBar';
+import SearchBar from 'app/components/events/searchBar';
+// import SmartSearchBar from 'app/components/smartSearchBar';
 import {IconAdd, IconDelete} from 'app/icons';
 import {t} from 'app/locale';
 import space from 'app/styles/space';
-import {Tag} from 'app/types';
+import {Organization, Project} from 'app/types';
 import Field from 'app/views/settings/components/forms/field';
 import SelectField from 'app/views/settings/components/forms/selectField';
 
-import {Aggregation, metricGroupByOptions, metricTags} from '../utils';
-
-type MetricQuery = {
-  tags: string;
-  groupBy: string;
-  aggregation: string;
-};
+import {Metric, MetricQuery} from './types';
 
 type Props = {
+  organization: Organization;
+  projectId: Project['id'];
+  metrics: Metric[];
   queries: MetricQuery[];
   onRemoveQuery: (index: number) => void;
   onAddQuery: () => void;
-  onChangeQuery: (queryIndex: number, metricQuery: MetricQuery) => void;
+  onChangeQuery: (queryIndex: number, query: MetricQuery) => void;
+  metric?: Metric;
 };
 
-function Queries({queries, onRemoveQuery, onAddQuery, onChangeQuery}: Props) {
+function Queries({
+  organization,
+  projectId,
+  metrics,
+  queries,
+  onRemoveQuery,
+  onAddQuery,
+  onChangeQuery,
+  metric,
+}: Props) {
   function handleFieldChange(queryIndex: number, field: keyof MetricQuery) {
     const widgetQuery = queries[queryIndex];
-
     return function handleChange(value: string) {
       const newQuery = {...widgetQuery, [field]: value};
       onChangeQuery(queryIndex, newQuery);
     };
   }
 
-  function getTagValue({key}: Tag, _query: string, _params: object) {
-    return Promise.resolve([metricTags[key as keyof typeof metricTags].value]);
-  }
+  // // this is just a mock
+  // function fetchTagValues(tagKey: string) {
```

Comment:
Should we remove these comments (and import ones) before merging?

Reply:
sure thing!